### PR TITLE
use handlebars to change css class of the currently opened folder

### LINKED ISSUE #461: Current folder highlight is removed when opening another unread mail
1. Go to your inbox, wait until the current message is loaded
2. Click on another _unread_ message
3. Observe how the highlight of the Inbox entry in the sidebar disappears

@DeepDiver1975 @wurstchristoph any idea?

### PR DESCRIPTION
**Pull request by wurstchristoph**
fixes #461 
@jancborchardt

### CODE REVIEW

File: js/mail.js [hunk 1]
```diff
@@ -652,10 +652,19 @@ var Mail = {
...
-			$('.mail_folders li')
-				.removeClass('active');
-			$('.mail_folders[data-account_id="' + accountId + '"] li[data-folder_id="' + folderId + '"]')
-				.addClass('active');
+			// disable all other folders for all accounts
+			_.each(Mail.State.accounts, function (account) {
+				var localAccount = Mail.State.folderView.collection.get(account.accountId);
+				var folders = localAccount.get('folders');
+				_.each(folders.models, function(folder) {
+					folders.get(folder).set("active", false);
```
Comment (jancborchardt) [inline js/mail.js]: code style: please use single apostrophes in JS ;)

File: js/mail.js [hunk 2]
```diff
@@ -652,10 +652,19 @@ var Mail = {
...
+				});
+			});
+
+			Mail.State.folderView.collection.get(accountId)
+				.get('folders')
+				.get(folderId)
+				.set("active", true);
```
Comment (jancborchardt) [inline js/mail.js]: Same here: please use 'single apostrophes' in JS ;)

**Comment by wurstchristoph** [comment]
@jancborchardt quotes replaced.

Btw. there are 60 " in mail.js :see_no_evil:

**Comment by jancborchardt** [comment]
@wurstchristoph thanks! This works very nicely and fixes #461 for me! A second review please @LukasReschke @DeepDiver1975 @zinks- @irgendwie! :)

(@wurstchristoph yeah, good reason to not introduce any more. ;) Some of them are there for a reason, because of CSS or escaping though.)

**Comment by jancborchardt** [comment]
@rullzer @zinks- a review here as well please. :)

**Comment by DeepDiver1975** [comment]
looks good :+1:

**Pull request merged**